createrepo_c.spec

```diff
@@ -72,8 +72,8 @@ Requires: rpm >= 4.9.0
 BuildRequires:  drpm-devel >= 0.1.3
 %endif
 
-%if 0%{?fedora} > 30 || 0%{?rhel} > 7
-Obsoletes:      createrepo < 0.11.0
+%if 0%{?fedora} || 0%{?rhel} > 7
+Obsoletes:      createrepo < %{version}-%{release}
```

Comment:
This is actually not in line with the Fedora Packaging Guidelines:

> $obsEVR is an (Epoch-)Version-Release tuple arranged so that there is a clean upgrade path but without gratuitously polluting the version space upwards. You usually do not use macros for this as you’re simply trying to advance beyond the last known release under the old name.

It's consistent with yum-3 stuff, but I don't see any other reason why it should be a macro.

Reply:
You're right. I've dropped this commit, thanks!